src/blog/2024/04/dashboard-1-8-0.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Dashboard 2.0: Milestones, PWA and New Components"
```

Comment:
Is PWA a well enough known term? Should we expand on it in the title?

Reply:
Not sure about the v1.8 piece - it's already confusing enough with 2.0 being referenced, and I've not explicitly called out version numbers in the titles before

Reply:
"New Components" makes the title more clickbait-y and more likely to have users click to find out what the components are, if we put too much into the title, no reason to engage

Reply:
Can you move the file to a path that matches the title at least?